doc/input-output-reference/src/overview/group-simulation-parameters.tex

```diff
@@ -790,6 +790,21 @@ \subsubsection{Inputs}\label{inputs-15-014}
   2;                       !- Maximum Number of HVAC Sizing Simulation Passes
 \end{lstlisting}
 
+\subsection{PerformancePrecisionTradeoffs}\label{performanceprecisiontradeoffs}
+
+The PerformancePrecisionTradeoffs object is to provide users with options for performance over precision tradeoffs for certain EnergyPlus features. This object enables users to choose certain options that speed up EnergyPlus simulations, but may lead to small decreases in the accuracy of results. 
+
+\paragraph{Field: Use Coil Direct Solutions}\label{use-coil-direct-solutions}
+
+If Yes, an analytical or empirical solution will be used to replace iterations in the coil performance calculations.
+
+An IDF example:
+
+\begin{lstlisting}
+PerformancePrecisionTradeoffs,
+  Yes;                      !- Use Coil Direct Solutions
+\end{lstlisting}
```

Comment:
This documentation does not cover the A2 field `Use Caching In Utility Functions`

Reply:
@Myoldmopar Thanks. We decided to get rid of the A2 field for now. Please see the latest commit.

Reply:
:+1: